src/engine/Drawing/SpriteSheet.ts

```diff
@@ -222,7 +307,7 @@ export class SpriteSheet extends Configurable(SpriteSheetImpl) {
 export class SpriteFontImpl extends SpriteSheet {
   private _currentColor: Color = Color.Black;
   private _currentOpacity: Number = 1.0;
-  private _sprites: { [key: string]: Sprite } = {};
+  private _spriteRecord: Record<string, Sprite> = {};
```

Comment:
21st century TS 💯

Reply:
We could consider a `Map` instead at some point depending on performance test.

Reply:
One cool this with a map, is you aren't restricted to just string keys